CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## main
 
+## 3.0.0
+
+- Introduced support for OTP Logger
+- Remove support for lager.
```

Comment:
Should we leave it there as "deprecated", and remove it in 4.x?

Reply:
It's been marked as deprecated since RC1 and planning to remove it already for the final release, WDYT? It's not tested and I very much doubt anybody's using it, lager was killed way too many years ago already.

Reply:
Ok, if it was marked as deprecated, I am good to go.